IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -377,6 +378,17 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if([navigationAction navigationType] == WKNavigationTypeLinkActivated) {
+        //Open secure web links with target=new window in default browser
+        if(([requestURL.scheme.lowercaseString isEqualToString:@"https"] && !navigationAction.targetFrame.isMainFrame) ||
```

Comment:
nit: add a space between if and the brace "if ("

Reply:
Addressed